batch/batch/worker/worker.py

```diff
@@ -2509,6 +2668,8 @@ def __init__(self, client_session: httpx.ClientSession):
 
         self.headers: Optional[Dict[str, str]] = None
 
+        self.cloudfuse_mount_manager = ReadOnlyCloudfuseManager()
```

Comment:
Do we need a shutdown method for the mount_manager?

Reply:
I don't think so. When the worker exits, all jobs that were running on the worker should have gone through their `cleanup` step and unmounted their respective buckets, so there should be no remaining work for the `ReadOnlyCloudfuseManager` to do.